src/map/clif.c

```diff
@@ -22215,6 +22215,104 @@ static bool clif_enchant_equipment(struct map_session_data *sd, enum equip_pos p
 #endif
 }
 
+static bool clif_lapineDdukDdak_open(struct map_session_data *sd, int itemId)
```

Comment:
Don't use camel case, please change itemId to itemid and function name all to lowercase.